Makefile

```diff
@@ -306,11 +309,11 @@ $(PKG): release-only
 	rm -rf $(PKGDIR)
 	rm -rf out/deps out/Release
 	$(PYTHON) ./configure --download=all --with-intl=small-icu \
-		--without-snapshot --dest-cpu=ia32 --tag=$(TAG)
```

Comment:
They are fixed in all io.js versions, but not 0.12, is my understanding. See this maybe: https://github.com/nodejs/io.js/pull/1663